### DISCUSSION TITLE
How to make local.ai run on windows

### DISCUSSION THREAD

**Discussion by Drete457** [louisgv/local.ai#110] [Q&A] [answered]
Hi, I installed the version v0.6.5 on my windows machine

I have windows 11 with, processor	12th Gen Intel(R) Core(TM) i7-1255U  1.70 GHz, RAM 32.0 GB, I download the model Dolly V2 3B press start, when I go to the browser and type localhost:8000 I only get a page saying pong.

Reading the documentation I don't see any extra step, any ideia what I'm missing?

I have both:
NodeJs  V20
Ruts V1.69
pnpm V8

**Comment by louisgv** [comment]
The completions endpoint is under `/completions`: https://github.com/louisgv/local.ai/wiki/API-with-cURL

**Reply by louisgv** [reply]
I think a lot of ppl missed this lol... perhaps we need to add docs on this to make it more clear maybe in the FAQ section

**Reply by Drete457** [reply]
I understand this answer, but is not working on the project's own UI, as shown in the videos.

**Reply by louisgv** [reply]
Oh after you pressed start server, did you make a new thread as well? (There should be a thread side-bar). That's where you will be able to take note + inferencing with the model

**Reply by Drete457** [reply]
Yes, I start and yes I press the new thread and try send message there to.

**Reply by louisgv** [reply]
Did you use `Shift + Enter`? - ref: https://github.com/louisgv/local.ai/issues/87 😅 ... <- will fix it soon lol. I guess what's fun for me to use is unintuitive for other xD... (tho there are time where it's not as useful :d..)

Perhaps a toggle might be better than having 2 buttons...

**Reply by Drete457** [reply]
Thank you for the answer, I'm starting to think I can't figure out this works, using the ctrl+enter gives some answer, but never answers directly what I ask, I'm missing something, and I'm unable to make work like the video.

**Reply by louisgv** [reply]
Oh, in the vid I used Wizard model - try that. Dolly 3B is not a very smart model lol..

**Reply by Drete457** [reply, marked as answer]
I understand what was missing, write the prompt then press ctrl+enter.... thank you for all the help
